extensions/wikia/AdEngine/js/spec/lookup/adapter/appnexus.spec.js

```diff
@@ -1,50 +1,56 @@
 describe('ext.wikia.adEngine.lookup.adapter.appnexus', function () {
 	'use strict';
 
-	function getAppNexus() {
-		return modules['ext.wikia.adEngine.lookup.adapter.appnexus'](
-			mocks.geo,
-			mocks.instantGlobals
-		);
-	}
-
 	var mocks = {
 		instantGlobals: {
 			wgAdDriverAppNexusBidderCountries: ['PL']
 		},
 		geo: {
 			isProperGeo: jasmine.createSpy('isProperGeo')
+		},
+		appNexusPlacements: {
+			getPlacement: function () {
+				return '123';
+			}
 		}
 	};
 
+	function getAppNexus() {
+		return modules['ext.wikia.adEngine.lookup.adapter.appnexus'](
+			mocks.geo,
+			mocks.instantGlobals,
+			mocks.appNexusPlacements
+		);
+	}
+
 	it('isEnabled checks the countries instant global', function () {
 		var appNexus = getAppNexus();
 		appNexus.isEnabled();
 		expect(mocks.geo.isProperGeo).toHaveBeenCalledWith(['PL']);
 	});
 
-	it('getAdUnits returns adUnits correctly', function() {
+	it('getAdUnits returns adUnits correctly', function () {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)